docs/en/04-supported-codebases/mmaction2.md

```diff
@@ -79,7 +79,7 @@ mim download mmaction2 --config tsn_r50_1x1x3_100e_kinetics400_rgb --dest .
 
 # convert mmaction2 model to onnxruntime model with dynamic shape
 python tools/deploy.py \
-    configs/mmaction/video-recognition/video-recognition_2d_onnxruntime_static.py \
+    configs/mmaction/video-recognition/video-recognition_onnxruntime_static.py.py \
```

Comment:
@irexyc still missing yml file for mmaction in https://github.com/irexyc/mmdeploy/tree/test-mmaction/tests/regression